### PR TITLE
Strip spaces from keywords and normalize keyword parsing

### PR DESCRIPTION
**Pull request by DragonRider0o0**

### CODE REVIEW

File: claat/util/util.go [hunk 1]
```diff
@@ -30,3 +30,28 @@ func Unique(a []string) []string {
...
+	}
+	s = stripSpaces(s)
+	strs := strings.Split(s, ",")
+	for k, v := range strs {
+		strs[k] = strings.ToLower(v)
+	}
+	return Unique(strs)
+}
+
+func stripSpaces(s string) string {
```
Comment (cassierecher) [inline claat/util/util.go:48]: Could we make use of http://go/godoc/strings/#TrimSpace instead?

File: claat/util/util_test.go [hunk 2]
```diff
@@ -37,3 +37,55 @@ func TestUnique(t *testing.T) {
 		})
 	}
 }
+
+func TestNormalizedSplit(t *testing.T) {
```
Comment (cassierecher) [inline claat/util/util_test.go:41]: Add a test case with some Unicode-defined spaces that are not U+0020

Comment (cassierecher) [inline claat/util/util_test.go:41]: Add a test case with multiple spaces consecutively

**Review by cassierecher — DISMISSED** [review]

File: claat/util/util.go (hunk 1 above)
Comment (DragonRider0o0) [reply claat/util/util.go:48]: No, we cannot.  We need to remove all of the spaces, not just leading and trailing.

File: claat/util/util_test.go (hunk 2 above)
Comment (DragonRider0o0) [reply claat/util/util_test.go:41]: Done

Comment (DragonRider0o0) [reply claat/util/util_test.go:41]: Done

**Review by shawnbuso — APPROVED** [review]
